kymatio/scattering2d/tests/test_scattering2d.py

```diff
@@ -1,11 +1,11 @@
 """ This script will test the submodules used by the scattering module"""
 
 import os
+import io
 import numpy as np
 import torch
 import pytest
 from kymatio.scattering2d import Scattering2D
-from kymatio.scattering2d import backend
```

Comment:
Why was this here? Leftover from before?

Reply:
yes, I realized one can remove it!

Reply:
Ok. Will move it into a separate commit.